src/client/entity-editor/validators/creator.js

```diff
@@ -49,8 +50,16 @@ export function validateCreatorSectionEndArea(value: any): boolean {
 
 export function validateCreatorSectionEndDate(
 	beginValue: any, endValue: any
-): boolean {
-	return validateDate(endValue) && dateIsBefore(beginValue, endValue);
+): object {
+	const {isValid, errorMessage} = validateDate(endValue);
+
+	if (isValid) {
+		if (dateIsBefore(beginValue, endValue)) {
+			return {errorMessage: '', isValid: true};
+		}
+		return {errorMessage: 'Date of death must be greated than Date of birth', isValid: false};
```

Comment:
It would be best here to use the `labelsForCreator` helper function, which means you'll have to pass a new `isGroup` argument to `validateCreatorSectionEndDate`.
See how it is used here:
https://github.com/bookbrainz/bookbrainz-site/blob/9e843b78a86e93a27363ff6db1125e29a69fb348/src/client/components/pages/entities/creator.js#L45-L51

Reply:
This is still to do, should be a quick fix.
<img width="564" alt="Capture d’écran 2019-05-27 à 19 51 07" src="https://user-images.githubusercontent.com/6179856/58433791-030d0200-80b9-11e9-820d-3aa3fd2257b3.png">

Reply:
I tried to fix this, it is not easy because of tests reusability, Please suggest generic error msg which will work for both type of author person and group.

Reply:
I took the liberty of pushing the code changes for this. I didn't run into any issue with the tests; what issue did you encounter?

Reply:
`validateCreatorSectionEndDate`  function is also called from [testValidateEndDateFunc](https://github.com/bookbrainz/bookbrainz-site/blob/2f081a0de9128a3cb3d601c88a70a8ab69f655bf/test/src/client/entity-editor/validators/helpers.js#L145) as `endDateValidationfunc`. `testValidateEndDateFunc`  function is common for author and publisher also. If intruduce a new argumenet `isGroup` in  `validateCreatorSectionEndDate` then how will we call `endDateValidationfunc` for both author and publisher?

Reply:
The code I pushed  does not require `authorType` to be defined in [validateAuthorSectionEndDate](https://github.com/bookbrainz/bookbrainz-site/pull/267/commits/8f7aed3a8cbf47bf9c030305dc1523d443ad28bf#diff-ad68a18cccd05bafe91d57126c7f7ce3R59), and falls back graciously, so there's no need to pass it in testValidateEndDateFunc.
We don't test for the exact message, so it does not make any difference for those tests.

Reply:
Thanks for the help.